receiver/jaegerreceiver/jaeger_agent_test.go

```diff
@@ -191,11 +191,20 @@ func testJaegerAgent(t *testing.T, agentEndpoint string, receiverConfig *configu
 	jr := newJaegerReceiver(jaegerAgent, receiverConfig, sink, set)
 	t.Cleanup(func() { require.NoError(t, jr.Shutdown(context.Background())) })
 
-	assert.NoError(t, jr.Start(context.Background(), componenttest.NewNopHost()), "Start failed")
+	var err error
+	for i := 0; i < 3; i++ {
+		err = jr.Start(context.Background(), componenttest.NewNopHost())
+		if err == nil {
+			break
+		} else {
+			time.Sleep(50 * time.Millisecond)
+		}
+	}
+	require.NoError(t, err, "Start failed")
```

Comment:
Hm.. didn't catch this. Looks like unrelated?

Reply:
Oops, sorry, made this branch on top of another PR. Sorry about that.